tutugodfrey/web-starter-godfrey
# #163449342 Add a map to the desktop version of the site

#### What does this pull request do?
- Add a map to the desktop version of the app.

#### Description of tasks to be completed
- Ensure the map render only on large screen
- Ensure markers are displayed for all restaurants
- Users can see the title of the restaurant when the markers are clicked

#### How should this be manually tested
- Make a copy of the branch to your local machine
- create a .dotenv file and add the environment variable REACT_APP_MAP_API_KEY as specified in .env.example file in your .env file. This should be the google maps API_KEY
- Run `npm install` to install dependency and `npm start` to start the app.
- After you add the API_KEY to .env, you may need to run cat .env in your terminal or you can manually export the variable in your terminal before starting the application.
- Confirm that the map is local and defaulting to Chicago and show markers for all restaurants

#### Relevant Pivotal Tracker story
#163449342
